### PR TITLE
{Small Fix] Add Lecture video url feild in lecture schema

### PR DESCRIPTION
**Pull request by Devesh21700Kumar**
**Description**

Add Lecture video url feild in lecture schema

**Type of Change:**

- [x] Code
- [x] New Feature
- [x] Documentation

**How Has This Been Tested?**

Terminal commands to test code:-
1). npm run lint
2). npm run lint:fix
3). nest start
4). npm run test

**Checklist**

- [x] My code follows the code style of this project.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] My changes generate no new warnings
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Review by Devesh21700Kumar — COMMENTED** [review]
Add Lecture video url in lecture schema for videos

File: src/modules/course/dto/update-lecture.dto.ts [hunk 1]
```diff
@@ -24,4 +24,13 @@ export class UpdateLectureDto {
...
   time: string;
+
+  /**
+   * The url video
+   * @example "https://codeforcause.org/video/1"
+   */
+  @IsNotEmpty()
+  @IsString()
+  @IsUrl()
+  lectureVideoUrl: string;
```
Comment (Abhishek-kumar09) [inline src/modules/course/dto/update-lecture.dto.ts:35]: This is update endpoint, I mean any field could remain unchanged at instance, for example while creating a lecture name has misspelled so only name need to be changed and not other properties, so don't impose isNotEmpty an all properties, instead add isOPtional.

Comment (Devesh21700Kumar) [reply src/modules/course/dto/update-lecture.dto.ts:35]: ok

Comment (Devesh21700Kumar) [reply src/modules/course/dto/update-lecture.dto.ts]: @Abhishek-kumar09 fixed

**Pull request merged**